### DISCUSSION TITLE
Conflict between custom config and preset with prefix

### DISCUSSION THREAD

**Discussion by bastienterrier** [tailwindlabs/tailwindcss#6126] [Help]
Hi community!
I'm currently using a `preset` that comes with a `vtmn-` prefix. My problem is when I want to extend the tailwind theme configuration with custom `height` for example, theses custom utilities also have the `vtmn-` prefix but I don't want too: theses classes have nothing to do with the `preset` I imported.
My question is: Is-it possible to remove the `vtmn-` prefix for custom theme extensions.

Here's a part of my Tailwind configuration:
```js
module.exports = {
  presets: [VitaminPreset], // comes with `vtmn-` prefix
  theme: {
    extend: {
      // Custom
      height: {
        'header-icon': `${HEADER_ICON_SIZE}px`, // generates `vtmn-h-header-icon` instead of `h-header-icon`
      },
    },
  },
};
```
Thank's a lot for your help :)

**Comment by Diogomartf** [comment]
Also faced the same problem. 

I had to replicate the theme extend config from the preset on the current tailwind config.

Would be great if tailwind added support for this.

**Comment by YannDuv** [comment]
I came to having a similar need. Does any new solution was brought during past year?
